### DISCUSSION TITLE
Note for M2 Mac users

### DISCUSSION THREAD

**Discussion by Neuronanc** [talmolab/sleap#1178] [General]
To download SLEAP, it is necessary to follow the M1 Mac instructions.

**Comment by gitttt-1234** [comment]
Closing this discussion, our installation documentation has been updated since this was posted and now covers Apple Silicon Macs (M1/M2/M3/M4) more explicitly. Check out the latest installation guide at [docs.sleap.ai](https://docs.sleap.ai/latest/installation/) for the most up-to-date instructions.
